### DISCUSSION TITLE
Pls create 2 turrets

### DISCUSSION THREAD

**Discussion by letovskiiy** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1424] [General]
Hi
Please create 2 turrets (multiple launch rocket system) add 40 artillery shells there instead of 6 as shown in the picture, and name it the MLRS "Vampire", and so that the projectiles throw cassettes.
![image](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/160428452/7d3bee65-d1ee-489e-8165-81dadce718c0)

**Comment by JameH2** [comment]
for once again i have to urge you that *suggestions are not allowed here* and from the way you put it we are not your personal butlers

if you *really* want these things in the mod, this can help you
https://www.w3schools.com/java/

**Reply by letovskiiy** [reply]
I will support you! I can support you financially.
And how did I put it? I'm just offering you new, cool ideas on how to make this best mod even better and more interesting, there's nothing wrong with that, these are just suggestions, I don't consider you personal butlers, these are just brilliant ideas. I know it's not easy to implement everything, but I think those who use the mod will fully support me.

**Reply by JameH2** [reply]
Thats great but your wording is basically 
“Hello i would like ONE artillery shell with a side of MLRS please 4x4 animal style, flip on one side simmer with phosgene shells, burn it then let it swim,” and then we are like “yes sir your total is 5 hours development time would you like to add a gratuity” 

Fyi there is no message that says “ Here at NTM industries we make sure our customers are satisfied”

Look, im sure your ideas are great but not only is bob busy basically with a TON of stuff (power net anyone?) theres also the fact that it is a *rule* that we dont allow suggestions and we WILL reject them no matter how popular or infamous they are

Sorry man
